modules/ting/ting.module

```diff
@@ -230,13 +203,18 @@ function ting_block_info() {
  * Implements hook_cron().
  *
  * Ensures that all expired entries are delete form the cache on cron runes.
+ * TODO BBS-SAL BBS-176: move/copy to opensearch as cache_opensearch
```

Comment:
Keep ting cache for now (see comment about this). So remove @todo.

Reply:
:+1: